content/en/platform/corda/4.8/open-source/token-sdk-introduction.md

```diff
@@ -54,7 +54,7 @@ Overview of changes:
 
```

Comment:
it's infinitely easier to fix this stuff as a suggested change if possible, rather than hunting down line numbers. In this instance, neither of these lines are headers in my version (possibly due to other merged changes since your review...)

Reply:
These still need fixing.